Packs/MitreCaldera/Integrations/MitreCaldera/README.md

```diff
@@ -0,0 +1,2444 @@
+| MitreCaldera.Operations.host_group.deadman_enabled | Boolean |  | 
+| MitreCaldera.Operations.host_group.ppid | Number |  | 
+| MitreCaldera.Operations.host_group.pid | Number |  | 
+| MitreCaldera.Operations.host_group.created | String |  | 
+| MitreCaldera.Operations.host_group.origin_link_id | String |  | 
+| MitreCaldera.Operations.host_group.last_seen | String |  | 
+| MitreCaldera.Operations.host_group.platform | String |  | 
+| MitreCaldera.Operations.host_group.pending_contact | String |  | 
+| MitreCaldera.Operations.host_group.host | String |  | 
+| MitreCaldera.Operations.host_group.group | String |  | 
+| MitreCaldera.Operations.host_group.location | String |  | 
+| MitreCaldera.Operations.host_group.display_name | String |  | 
+| MitreCaldera.Operations.host_group.upstream_dest | String |  | 
+| MitreCaldera.Operations.host_group.sleep_max | Number |  | 
+| MitreCaldera.Operations.host_group.architecture | String |  | 
+| MitreCaldera.Operations.host_group.sleep_min | Number |  | 
+| MitreCaldera.Operations.host_group.server | String |  | 
+| MitreCaldera.Operations.host_group.contact | String |  | 
+| MitreCaldera.Operations.host_group.privilege | String |  | 
+| MitreCaldera.Operations.host_group.username | String |  | 
+| MitreCaldera.Operations.host_group.trusted | Boolean |  | 
+| MitreCaldera.Operations.host_group.proxy_chain | String |  | 
+| MitreCaldera.Operations.host_group.paw | String |  | 
+| MitreCaldera.Operations.host_group.exe_name | String |  | 
+| MitreCaldera.Operations.obfuscator | String |  | 
+| MitreCaldera.Operations.use_learning_parsers | Boolean |  | 
+| MitreCaldera.Operations.jitter | String |  | 
+| MitreCaldera.Operations.start | String |  | 
+| MitreCaldera.Operations.auto_close | Boolean |  | 
+
+### caldera-create-objective
+***
+Create a new objective
+
+
+#### Base Command
+
+`caldera-create-objective`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| name | Objective name. | Optional | 
+| goals | Goals (array of objective objects). | Optional | 
+| description | Description. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| MitreCaldera.Objectives.name | String |  | 
+| MitreCaldera.Objectives.id | String |  | 
+| MitreCaldera.Objectives.percentage | Unknown |  | 
+| MitreCaldera.Objectives.goals.count | Number |  | 
+| MitreCaldera.Objectives.goals.achieved | Boolean |  | 
+| MitreCaldera.Objectives.goals.operator | String |  | 
+| MitreCaldera.Objectives.goals.value | String |  | 
+| MitreCaldera.Objectives.goals.target | String |  | 
+| MitreCaldera.Objectives.description | String |  | 
+
+### caldera-create-relationship
+***
+Create a Relationship
+
+
+#### Base Command
+
+`caldera-create-relationship`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| relationship_unique | . | Optional | 
```

Comment:
There is about 178 descriptions that are just `| . |`
if the description are not required - just leave them empty